package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/eth-block-tracker",
-  "version": "11.0.0",
+  "version": "11.0.1",
```

Comment:
any reason why are we making a patch release ? not a minor ?

Reply:
This is the only PR included in this release: https://github.com/MetaMask/eth-block-tracker/pull/261

Is there any noteworthy feature being added in those version bumps?

I checked that the utils methods with breaking changes aren't used in this package.